RELEASE.md

```diff
@@ -1,13 +1,20 @@
 # Release steps
 
-* Ensure master passes CI tests
-* Bump version in package.json. Any breaking change or new feature should bump
-  minor (or even major). Non-breaking changes or fixes can just bump patch.
-* Update README manually if the changes are not documented in-code. Run
+1. Ensure master passes CI tests
+1. Bump version:
+  - `$ npm version <major|minor|patch>`
+
+    >`major` - breaking API changes  
+    >`minor` - backwards-compatible features  
+    >`patch` - backwards-compatible bug fixes  
```

Comment:
Removed previous manual package bumping and committing instructions for `npm version` command.  It bumps package.json, creates tags, and bump commit.

Also, the version suggestions were ambiguous and not semver.org compliant.  Updated to match the [semver](semver.org) specs.